data/korath/korath ships.txt

```diff
@@ -8,11 +8,109 @@
+	attributes
+		category "Heavy Freighter"
+		"cost" 29370000
+		"shields" 39000
+		"hull" 16000
+		"required crew" 191
+		"bunks" 395
+		"mass" 1503
+		"drag" 18.3
+		"heat dissipation" .65
+		"fuel capacity" 900
+		"cargo space" 605
+		"outfit space" 827
+		"weapon capacity" 297
+		"engine capacity" 181
+		
+		weapon
+			"blast radius" 300
+			"shield damage" 4600
+			"hull damage" 2300
+			"hit force" 5900
+	outfits
+		"Korath Banisher" 2
+		"Korath Grab-Strike" 3
+		"Korath Warder" 2
+		
+		"Systems Core (Large)"
+		"Triple Plasma Core"
+		"Large Heat Shunt" 3
+		"Fuel Processor"
+		"Korath Repeater Rifle" 190
+		
+		"Steering (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Thruster (Asteroid Class)"
+		"Hyperdrive"
+		"Jump Drive"
+		
+	turret -37 -155 "Korath Banisher"
+	turret 0 -155 "Korath Warder"
+	turret 37 -155 "Korath Grab-Strike"
+	turret -96 0 "Korath Grab-Strike"
+	turret 96 0 "Korath Grab-Strike"
+	turret -80 130 "Korath Warder"
+	turret 80 130 "Korath Banisher"
+	engine -33 171 .4
+	engine -2 182 .6
+	engine 31 171 .4
+	fighter 51 59
+	fighter -51 59
+	fighter 0 -42
+	fighter 0 16
+	explode "tiny explosion" 120
+	explode "small explosion" 60
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 15
+	"final explode" "final explosion large"
+	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and Raiders lacked the cargo capacity. The Dredger was born as an answer to this problem, featuring an abundance of cargo and passenger space, with formidable defensive armament. With such a complement, these bulky ships can also offer potent support for frontline Raiders."
+
+
+
+ship "Korath Dredger" "Korath Dredger (Crippled)"
+	sprite "ship/dredger"
+	outfits
+		"Korath Banisher" 2
+		"Korath Grab-Strike" 1
+		"Korath Warder" 1
+		
+		"Triple Plasma Core"
+		"Large Heat Shunt" 3
+		"Fuel Processor"
+		"Korath Repeater Rifle" 190
+		
+		"Steering (Planetary Class)"
+		"Steering (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Thruster (Comet Class)"
+		"Hyperdrive"
+		"Jump Drive (Broken)"
+		
+	turret -37 -155 "Korath Banisher"
+	turret 0 -155 "Korath Warder"
+	turret 37 -155
+	turret -96 0
+	turret 96 0 "Korath Grab-Strike"
+	turret -80 130
+	turret 80 130 "Korath Banisher"
+	description "After the Korath were exiled, efficient resource collection became a priority, particularly through raiding. Their world-ships were too precious to risk on dangerous forays, and Raiders lacked the capacity. The Dredger was their first new ship: abundant cargo and passenger space, bays for four fighters, and seven turrets. With such a complement, these bulky ships are formidable supports for frontline Raiders."
+	description "This Dredger has taken significant damage, and is missing several key systems, such as its system core and some of its weaponry."
```

Comment:
Even if ship descriptions become visible on ships the player owns, this description would stop making sense as soon as the player changes the outfits in any way.

Reply:
Yeah, that's true. I'll remove it.

I really wish we had some way to have variants have a different description in the shipyard while still having the normal description for the hull.